docs/theming.md

```diff
@@ -71,6 +71,15 @@ Themes can provide a set of custom MDX components, including a replacement for t
 
 MDX Deck includes two themes for adding syntax highlighting with [react-syntax-highlighter][]: `syntaxHighlighter` and `syntaxHighlighterPrism`.
 
+```js
+import { future } from "mdx-deck/themes";
+import { syntaxHighlighterPrism } from "mdx-deck/themes";
+
+export default syntaxHighlighterPrism({
+  ...future
+});
```

Comment:
Thanks for putting this together! I think the preferred API should be in the section above this: https://github.com/jxnblk/mdx-deck/blob/master/docs/theming.md#composing-themes

Reply:
There is no description at all currently of how this works. This example would have definitely saved me some time and frustration at the time. So agree that an even better description of the API above would be lovely. Don't think I'm the person to write it though (although I'd love to help), mainly because it's unclear how it does actually work...